README.hbs

```diff
@@ -1,10 +1,47 @@
 # node-sass-extra
 
-A drop-in replacement for [node-sass](https://github.com/sass/node-sass) that adds support for globs, promises and more.
-
 [![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg)](http://commitizen.github.io/cz-cli/)
+[![Open Source Love](https://badges.frapsoft.com/os/mit/mit.svg?v=102)](https://github.com/ellerbrock/open-source-badge/)
+[![Open Source Love](https://badges.frapsoft.com/os/v2/open-source.svg?v=102)](https://github.com/ellerbrock/open-source-badge/)
+[![GitHub issues](https://img.shields.io/github/issues/marksmccann/node-sass-extra)](https://github.com/marksmccann/node-sass-extra/issues)
```

Comment:
@phillipluther We still need to add badges for the build, coverage, version, and dependencies. Do you have some experience with that? It was a little more difficult that I anticipated. I think it was extra difficult because we have nothing published or on master. Maybe we open an issue to add those tags after release?

Reply:
If you wanted to add additional badges to this PR, feel free. :)